docs/release.md

```diff
@@ -0,0 +1,22 @@
+# Status AddOn Relesea process 
+
+## Steps to make release
+
+1. Checkout main and fetch from upstream
+2. git checkout <release branch> # e.g. release-0.2
+3. Rebase from main
```

Comment:
Maybe add the explicit command - people interpret "rebase from main" differently

Reply:
Why a rebase is needed here ?  didn't we start with latest main before the checkout ?

Reply:
> Maybe add the explicit command - people interpret "rebase from main" differently

Done

> Why a rebase is needed here ? didn't we start with latest main before the checkout ?

we updated main from upstream, but what about the release branch ? The tag for the release should be relative to the release branch; shouldn't the release branch be updated with fixes / updates that have been merged in main ?

Reply:
If we are ok to not using a release branch this point is moot.